README.md

```diff
@@ -84,71 +74,92 @@ git push -u origin $ASB_TEAM_NAME
 
 ```
 
-### Setup AKS Secure Baseline
+### Choose your deployment region
 
-> This section takes 45 - 60 minutes to complete
+```bash
 
-#### Initial Setup
+### choose the closest pair - not all regions support ASB
+export ASB_LOCATION=eastus2
+export ASB_GEO_LOCATION=centralus
 
-```bash
+export ASB_LOCATION=centralus
+export ASB_GEO_LOCATION=eastus2
 
-# validate team name is set up
-echo $ASB_TEAM_NAME
+export ASB_LOCATION=westus2
+export ASB_GEO_LOCATION=westcentralus
 
-# verify the correct subscription
-az account show
+export ASB_LOCATION=northeurope
+export ASB_GEO_LOCATION=westeurope
 
-# check certs
-echo $APP_GW_CERT
-echo $INGRESS_CERT
-echo $INGRESS_KEY
+export ASB_LOCATION=francecentral
+export ASB_GEO_LOCATION=francesouth
```

Comment:
The paired region docs says some of the regions like `francesouth` "are available only upon request by creating a new support request in the Azure portal". Is that a risk, or something to call out in this README?

https://docs.microsoft.com/en-us/azure/best-practices-availability-paired-regions#azure-regional-pairs

Reply:
@AAkindele - can you remove any paired regions with constraints and merge?